CHANGES.rst

```diff
@@ -9,8 +9,12 @@
 Changes
 =======
 
+Version 2.3.2 (released 2021-06-15)
+-----------------------------------
+
+- Remove invenio dependencies to depend only on rdm-records.
 
-Version 2.3.1 (released 2021-06-10)
+Version 2.3.1 (removed)
```

Comment:
@lnielsen is this correct? or how do we state that it was taken down from PyPi?

Reply:
I think it is ok like this, so we keep track of it...